RestaurantMap/Scener/Restaurant/RestaurantViewController.swift

```diff
@@ -25,8 +42,41 @@ class RestaurantViewController: UIViewController {
         presenter.router = router
         interactor?.presenter = presenter
     }
+
+    private func setupUI() {
+        guard let restaurant = restaurant else { return }
+        nameLabel.text = restaurant.name
+        handleRestaurantRate(rate: restaurant.rating)
+        reviewCountLabel.text = String(restaurant.reviewCount)
+        priceLabel.text = restaurant.price
+        handleDisplayAdress(address: restaurant.location.displayAddress)
+        imageView.setImage(url: restaurant.getImageURL(), placeholderImageName: "placeholder_icon")
+    }
+    
+    private func handleRestaurantRate(rate: Int) {
```

Comment:
Shouldn't declare this func here, can create a var as starsDisplay: String in Restaurant model